clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/a300/fan_control.hpp

```diff
@@ -0,0 +1,172 @@
+#ifndef CLEARPATH_HARDWARE_INTERFACES_A300_FAN_CONTROL_HPP
+#define CLEARPATH_HARDWARE_INTERFACES_A300_FAN_CONTROL_HPP
+
+#include <rclcpp/rclcpp.hpp>
+#include <sensor_msgs/msg/battery_state.hpp>
+#include <clearpath_platform_msgs/msg/temperature.hpp>
+#include <clearpath_platform_msgs/msg/fans.hpp>
+#include <clearpath_motor_msgs/msg/lynx_multi_status.hpp>
+#include <mutex>
+#include <map>
+
+namespace a300_cooling
+{
+/// Fan control topic name
+const std::string FAN_CONTROL_TOPIC_NAME = "platform/mcu/cmd_fans";
+
+/// MCU temperature topic name
+const std::string MCU_TEMPERATURE_TOPIC = "platform/mcu/status/temperature";
+
+/// Motor temperature topic name
+const std::string MOTOR_TEMPERATURE_TOPIC = "platform/motors/status";
+
+/// Battery temperature topic name
+const std::string BATTERY_TEMPERATURE_TOPIC = "platform/bms/battery_state";
+
+/// Fan command for high error state, percent of maximum speed
+constexpr float FAN_CMD_HIGH_ERROR = 1.0f;
+
+/// Fan command for high warning state, percent of maximum speed
+constexpr float FAN_CMD_HIGH_WARNING = 0.75f;
+
+/// Fan command for low error state, percent of maximum speed
+constexpr float FAN_CMD_LOW_ERROR = 0.0f;
+
+/// Fan command for low warning state, percent of maximum speed
+constexpr float FAN_CMD_LOW_WARNING = 0.25f;
+
+/// Fan command for normal state, percent of maximum speed
+constexpr float FAN_CMD_NORMAL = 0.25f;
```

Comment:
Just double-checking that FAN_CMD_NORMAL was intentionally set to the same value as FAN_CMD_LOW_WARNING.

Reply:
Fixed.